java/src/test/resources/testng.xml

```diff
@@ -3,6 +3,19 @@
 
 <suite name="ome.jxrlib">
 
+  <test name="FL_3channel_noZ_compression_lossless">
```

Comment:
It's the largest of the test files, so moving it was moved up in the hope that (combined with the buffer reallocation workaround in e29028c) running it first would avoid the issues with `ByteBuffer`s. (This worked on my machine, but apparently not on Travis.)